contracts/Pod.sol

```diff
@@ -582,6 +582,24 @@ contract Pod is
         return ticket.balanceOf(address(this));
     }
 
+    /**
+     * @notice Calculate underlying tokens via shares input.
+     * @dev Using shares as input calculate the underlying tokens
+     * @return Underlying tokens
+     */
+    function _calculateUnderlyingTokens(uint256 _shares)
+        internal
+        view
+        returns (uint256)
+    {
+        // Check totalSupply to prevent SafeMath: division by zero
+        if (totalSupply() > 0) {
```

Comment:
This is a good opportunity to pull totalSupply() into a variable so that there is only one SLOAD